diff -up plasma6-kwin-6.3.3/CMakeLists.txt.omv~ plasma6-kwin-6.3.3/CMakeLists.txt
--- plasma6-kwin-6.3.3/CMakeLists.txt.omv~	2025-03-11 20:52:02.588361718 +0100
+++ plasma6-kwin-6.3.3/CMakeLists.txt	2025-03-11 20:52:09.927919805 +0100
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.16)
 
-set(PROJECT_VERSION "6.3.2") # Handled by release scripts
+set(PROJECT_VERSION "6.3.3") # Handled by release scripts
 project(KWin VERSION ${PROJECT_VERSION})
 
 set(CMAKE_C_STANDARD 99)
