diff -up plasma6-kwin-6.3.3/tests/pointerconstraintstest.cpp.omv~ plasma6-kwin-6.3.3/tests/pointerconstraintstest.cpp
--- plasma6-kwin-6.3.3/tests/pointerconstraintstest.cpp.omv~	2025-03-14 02:33:34.869893657 +0100
+++ plasma6-kwin-6.3.3/tests/pointerconstraintstest.cpp	2025-03-14 02:34:06.171802037 +0100
@@ -369,7 +369,7 @@ int main(int argc, char **argv)
     QGuiApplication app(argc, argv);
 
     Backend *backend;
-    if (app.platformName() == QStringLiteral("wayland")) {
+    if (app.platformName().startsWith(QStringLiteral("wayland"))) {
         qDebug() << "Starting up: Wayland native mode";
         backend = new WaylandBackend(&app);
     } else {
