diff -urN kwin-5.3.0/scripts/minimizeall/metadata.desktop kwin-5.3.0-patched/scripts/minimizeall/metadata.desktop
--- kwin-5.3.0/scripts/minimizeall/metadata.desktop	2015-04-23 20:06:05.000000000 +1000
+++ kwin-5.3.0-patched/scripts/minimizeall/metadata.desktop	2015-05-02 01:34:17.156343645 +1000
@@ -40,6 +40,7 @@
 X-KDE-PluginInfo-Email=thomas.luebking@gmail.com
 X-KDE-PluginInfo-Name=minimizeall
 X-KDE-PluginInfo-Version=1.0
+X-KDE-PluginInfo-EnabledByDefault=true
 
 X-KDE-PluginInfo-Depends=
 X-KDE-PluginInfo-License=GPL
