--- kwin-5.10.3/logind.cpp.0000~	2017-06-27 11:52:14.000000000 +0200
+++ kwin-5.10.3/logind.cpp	2017-06-28 01:19:19.079487887 +0200
@@ -312,7 +312,7 @@ int LogindIntegration::takeDevice(const
                                                           m_sessionPath,
                                                           s_login1SessionInterface,
                                                           QStringLiteral("TakeDevice"));
-    message.setArguments(QVariantList({QVariant(major(st.st_rdev)), QVariant(minor(st.st_rdev))}));
+    message.setArguments(QVariantList({QVariant(gnu_dev_major(st.st_rdev)), QVariant(gnu_dev_minor(st.st_rdev))}));
     // intended to be a blocking call
     QDBusMessage reply = m_bus.call(message);
     if (reply.type() == QDBusMessage::ErrorMessage) {
@@ -333,7 +333,7 @@ void LogindIntegration::releaseDevice(in
                                                           m_sessionPath,
                                                           s_login1SessionInterface,
                                                           QStringLiteral("ReleaseDevice"));
-    message.setArguments(QVariantList({QVariant(major(st.st_rdev)), QVariant(minor(st.st_rdev))}));
+    message.setArguments(QVariantList({QVariant(gnu_dev_major(st.st_rdev)), QVariant(gnu_dev_minor(st.st_rdev))}));
     m_bus.asyncCall(message);
 }
 
